site/public/js/user-profile.js

```diff
@@ -58,6 +64,54 @@ function getCurrentUTCOffset() {
     return `${sign + hours}:00`;
 }
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
As per [these](https://submitty.org/developer/software_and_system_design/coding_style_guide/javascript#style-rules) docs, it would be better to add the line `/** exported updateUserPronouns */` to the top of this file.  It is unfortunate that the rest of this file does not follow this proper convention, but it would be good to follow the convention for these new changes.  Apply throughout the rest of the changes in this file.

Reply:
fixed

Reply:
This still hasn't been fixed.

Reply:
Should be fixed now